test/extraction_lambda/test_monitor.py

```diff
@@ -87,18 +87,20 @@ def test_get_current_state_returns_0_if_stats_json_missing_key(s3, monitor):
 
 
 def test_save_state_saves_new_state_to_s3_bucket(s3, monitor):
-    with patch('extraction.monitor.datetime') as mock_date:
-        mock_date.now.return_value = datetime.datetime(2023, 3, 27, 1, 2, 3, 4)
+    with patch('extraction.monitor.Monitor.get_utc_timestamp') as mock_timestamp:
+        # mock_date.now.return_value = datetime.datetime(
```

Comment:
This could work with a mock like this: datetime.now().replace().timestamp.return_value = ...